PocketKit/Sources/Sync/NSPersistentContainer+Extensions.swift

```diff
@@ -23,3 +23,11 @@ extension NSPersistentContainer {
         return container
     }
 }
+
+private class SharedContainer: NSPersistentContainer {
+    override class func defaultDirectoryURL() -> URL {
+        FileManager.default
+            .containerURL(forSecurityApplicationGroupIdentifier: "group.com.mozilla.pocket")!
+            .appendingPathComponent("pocket.sqlite")
+    }
+}
```

Comment:
This is what we need to do in order to access the same core data backing store (sqlite) from within the app group